Code/GraphMol/new_canon.h

```diff
@@ -813,8 +846,8 @@ RDKIT_GRAPHMOL_EXPORT void ActivatePartitions(unsigned int nAtoms, int *order,
 RDKIT_GRAPHMOL_EXPORT void rankMolAtoms(
     const ROMol &mol, std::vector<unsigned int> &res, bool breakTies = true,
     bool includeChirality = true, bool includeIsotopes = true,
-    bool includeAtomMaps = true, bool includeChiralPresence = false,
-    bool includeStereoGroups = true);
+    bool includeAtomMaps = true, bool useNonStereoRanks = false,
```

Comment:
Well, I did add it at the end.  In the meantime, someone else (probably you) added others too, so when it merged, this is what it did.     I had to traced all uses of this routine to make sure they had the correct calling args - I think they are alright.   This is a good reason to have a class of options. New additions are order independent.

Reply:
> Well, I did add it at the end. In the meantime, someone else (probably you) added others too, so when it merged, this is what it did

Be that as it may: you are changing the order of the arguments to a function which is in released code. We can't do that without a really good reason.

> This is a good reason to have a class of options. New additions are order independent.

I 100% agree

Reply:
OK I moved it.